lab-justin/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "lab-justin",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "test": "./node_modules/mocha/bin/mocha"
```

Comment:
One of the things I like to do is write a little alias for starting the database, like
```javascript
"scripts": {
"start:db": "mongod -dbpath=./db --smallfiles"
}
```